session-5-going-further (#472)
nfidd/nfidd: sessions/R-estimation-and-the-renewal-equation.qmd

---
title: "$R_t$ estimation and the renewal equation"

> bibliography: ../nfidd.bib

Reviewer: ```suggestion
```

Removing for now and need to add back in in #491

# Going further

## Challenge

- We have used symptom onsets under the assumption that every infected person develops symptoms. 
Earlier we also created a time series of hospitalisation under the assumption that only a proportion (e.g., 30%) of symptomatic individuals get hospitalised. 
How would you change the model in this case?
What are the implications for inference?

## Methods in practice

> - The `EpiEstim` software provides a range of accessible tools for estimating $R_t$, using a simpler approach than the model we have used here. Alternatively, the `EpiNow2` [@abbottEpiNow2EstimateRealtime2025] package implements a more complex version of the model we have used. Taking a different approach, `Epidemia` @bhattSemiMechanisticBayesianModeling2020 implements a regression model approach to $R_t$ estimation.

Reviewer: ```suggestion
- `EpiEstim` provides a range of accessible tools for estimating $R_t$, using a simpler approach than the model we have used here.
- `EpiNow2` [@abbottEpiNow2EstimateRealtime2025] package again implements a range of models similar to the model we have used here.
- `Epidemia` @bhattSemiMechanisticBayesianModeling2020 implements a regression model approach to $R_t$ estimation.
```

> - In this course we focused on the instantaneous reproduction number, which is most suitable for real-time analysis when not all infections have yet been observed. Methods to estimate the case reproduction number are implemented in e.g. @wallingaDifferentEpidemicCurves2004, and are most appropriate using retrospective data that represent a complete final count of an outbreak.

Reviewer: ```suggestion
- In this course we focused on the instantaneous reproduction number.
An alternative is the case reproduction number implemented in @wallingaDifferentEpidemicCurves2004.
```
This is often said but I think it's really quite a lot more nuanced than this so removing.
